src/pages/ChooseCubePage/ChooseCubePage.scss

```diff
@@ -20,7 +22,8 @@
 		display: flex;
 		flex-direction: column;
 		align-items: center;
-		padding: 20px;
+		padding: 30px;
+		color: rgb(182, 0, 182);
```

Comment:
Let's use scss variables or bootstrap classes for colors. We need a color scheme